h/services/search_index.py

```diff
@@ -230,24 +242,25 @@ def sync(self, limit):
         if annotation_ids_to_sync:
             self._batch_indexer.index(list(annotation_ids_to_sync))
 
+        if annotation_ids_to_delete:
+            self._batch_indexer.delete(list(annotation_ids_to_delete))
```

Comment:
So each run of  the `sync_annotations()` task now makes potentially two calls to Elasticsearch's `streaming_bulk()` method: one for annotations to create or update and another for annotations to delete. Due to batching each call to `streaming_bulk()` may make multiple requests to Elasticsearch's bulk API though in practice with our current parameters it'll always be only one request.